addons/purchase/tests/test_access_rights.py

```diff
@@ -49,6 +49,26 @@ def setUpClass(cls, chart_template_ref=None):
             'type': 'service',
         })
 
+        cls.purchase_order_1 = cls.env['purchase.order'].create({
+            "partner_id": cls.vendor.id,
+            "order_line": [
+                (0, 0, {
+                    'product_id': cls.product.id,
+                    'name': f'{cls.product.name} {1:05}',
+                    'price_unit': 79.80,
+                    'product_qty': 15.0}),
+                (0, 0, {
+                    'product_id': cls.product.id,
+                    'name': f'{cls.product.name} {2:05}',
+                    'price_unit': 286.70,
+                    'product_qty': 5.0}),
+                (0, 0, {
+                    'product_id': cls.product.id,
+                    'name': f'{cls.product.name} {3:05}',
+                    'price_unit': 99.00,
+                    'product_qty': 4.0}),
+            ]})
```

Comment:
Since `test_double_validation` only checks if an user can/cannot approve a purchase, we don't a to mimic the demo data purchase order and we can only do the minimum:
```suggestion
            "order_line": [(0, 0, {
                'product_id': cls.product.id,
                'name': cls.product.name,
                'price_unit': 79.80,
                'product_qty': 15.0,
            ]})
```

Reply:
Done @svs-odoo  , rebased and tests pass.